serverless-functions/.gitignore

```diff
@@ -1,2 +1,2 @@
 # VIDEO. IGNORE THE BUILT CUSTOMER FACING APP ASSETS
-src/assets/features/chat-to-video
\ No newline at end of file
+#src/assets/features/chat-to-video
```

Comment:
Rather than committing built assets, maybe the build should be part of CI?

Reply:
@dremin If we can already pre-built the "video-app-quickstart" web-app into the serverless functions as an asset, that would be even better.

Reply:
yeah absolutely, this is what i was going to do i was just capturing Leroys changes before i parked it to do something else last Friday